### PR TITLE
Pass in "image_name" arg from the pipeline as a string instead of a file

### PR DESCRIPTION
**Pull request by gargshobhit90**
JIRA: [STAB-441]

### CODE REVIEW

File: push_docker.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
-IMAGE_ID=$1
+response="$(docker build .)"
```
Comment (jeffwatts) [inline push_docker.sh:3]: What docker image are we now building which we weren't building before? We don't use the build in the Makefile?

Comment (jeffwatts) [reply push_docker.sh:3]: Never mind the first question -- I see that this is the concourse kube multi-deployment resource. Just wondering why we now build it in this script when we didn't before.

Comment (gargshobhit90) [reply push_docker.sh:3]: We used to still build the image by running `docker build .` in the local console. In essence, the script did half the job (just the pushing part) then. I'm bringing that into this script.

In fact, we do the exact same thing in the other concourse-kubernetes-resource and this same logic is already there as well.

Comment (jeffwatts) [reply push_docker.sh:3]: Ah, okay. I definitely have a preference for defining these kinds of things in Makefiles or similar build-specific tooling, which already seems to support building, tagging, and pushing, but that's a tiny nit and not really important right now with such as simple, infrequently changed repo.

**Review by jeffwatts — APPROVED** [review]

**Pull request merged**